packages/components/pie-switch/src/index.ts

```diff
@@ -197,7 +205,9 @@ export class PieSwitch extends RtlMixin(LitElement) implements SwitchProps {
                 class="c-switch-wrapper"
                 ?isRTL=${isRTL}
                 ?isDisabled=${isDisabled}>
-                ${labelPlacement === 'leading' ? this.renderSwitchLabel() : nothing}
+                <pie-form-label for="pie-switch">
```

Comment:
I only placed the form-label component in this template so the styling looked nicer. It might be preferable to place it next to this component in the consumer template. Either approach works the same so it is a matter of preference